tools/cockpit.spec

```diff
@@ -597,6 +597,8 @@ The Cockpit component for creating diagnostic reports with the
 sosreport tool.
 
 %files sosreport -f sosreport.list
+/usr/share/metainfo/org.cockpit-project.cockpit-sosreport.metainfo.xml
+/usr/share/pixmaps/cockpit-sosreport.png
```

Comment:
This needs to be guarded with `%if 0%{?rhel} == 0` as on RHEL sosreport is folded into cockpit-system:
```
# On RHEL kdump, subscriptions, networkmanager, selinux, and sosreport are part of the system package
%if 0%{?rhel}
cat kdump.list subscriptions.list sosreport.list networkmanager.list selinux.list >> system.list
%endif
```

Reply:
It is guarded already, no?  But we need to remove the metadata files.

Reply:
Indeed, it's guarded already. I would have thought that you needed to remove the files, otherwise rpm would croak about "uninstalled files". The currently running rhel* tests are still at package build stage though, so they didn't fail on this yet (or perhaps you already retried)

Reply:
> Indeed, it's guarded already. I would have thought that you needed to remove the files, otherwise rpm would croak about "uninstalled files".

Yes, that's what happened.  Should be fixed already.